copier/copier.go

```diff
@@ -1164,6 +1164,10 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 						}
 						symlinkTarget = target
 					}
+					if info.Mode()&os.ModeType == os.ModeSocket {
+						logrus.Debugf("Skipping socket %q", info.Name())
+						return nil
+					}
```

Comment:
Can/should we move this if check higher up to avoid some of the other processing above this?

Reply:
sure, moved